src/chrome/content/rules/Umsmash.com.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="Umsmash.com">
+	<target host="umsmash.com" />
+	<target host="www.umsmash.com" />
+	<rule from="^http:" to="https:" />
```

Comment:
I know this is minor, but for the sake of readability and continuity, please put a space between target and rules.

Reply:
Example: https://github.com/EFForg/https-everywhere/pull/19812/files